<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OFMagasinDialog</class>
 <widget class="QDialog" name="OFMagasinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horaires d'Ouverture et de Fermeture du magasin</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="OFInterfaceWidget" name="tabLundi">
      <attribute name="title">
       <string>Lundi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabMardi">
      <attribute name="title">
       <string>Mardi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabMercredi">
      <attribute name="title">
       <string>Mercredi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabJeudi">
      <attribute name="title">
       <string>Jeudi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabVendredi">
      <attribute name="title">
       <string>Vendredi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabSamedi">
      <attribute name="title">
       <string>Samedi</string>
      </attribute>
     </widget>
     <widget class="OFInterfaceWidget" name="tabDimanche">
      <attribute name="title">
       <string>Dimanche</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dayLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copieButton">
       <property name="text">
        <string>Copier le jour sur :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lundiCheck">
       <property name="text">
        <string>Lundi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mardiCheck">
       <property name="text">
        <string>Mardi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mercrediCheck">
       <property name="text">
        <string>Mercredi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="jeudiCheck">
       <property name="text">
        <string>Jeudi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="vendrediCheck">
       <property name="text">
        <string>Vendredi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="samediCheck">
       <property name="text">
        <string>Samedi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dimancheCheck">
       <property name="text">
        <string>Dimanche</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveCloseButton">
       <property name="text">
        <string>Enregistrer et fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fermer sans enregister</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exceptionButton">
       <property name="text">
        <string>Ouverture Exeptionnelle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OFInterfaceWidget</class>
   <extends>QWidget</extends>
   <header>ofinterfacewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>OFMagasinDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>OFMagasinDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
